Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2_test.py

```diff
@@ -107,3 +112,21 @@ def test_import_ioc_without_approval(mocker):
 
     assert results[0]['Contents']
     assert expected_import_json == http_mock.call_args[1]['json']
+
+
+@pytest.mark.parametrize('file_hash, expected_result, raw_response', [
+    ('178ba564b39bd07577e974a9b677dfd86ffa1f1d0299dfd958eb883c5ef6c3e1',
```

Comment:
Let's keep `178ba564b39bd07577e974a9b677dfd86ffa1f1d0299dfd958eb883c5ef6c3e1` as a const, so readers will know why it's here. Let's do this throughout the code